src/app/modules/mpi/components/paciente-detalle.component.ts

```diff
@@ -13,6 +13,8 @@ import { Auth } from '@andes/auth';
 })
 export class PacienteDetalleComponent implements OnInit, OnChanges {
     @Input() orientacion: 'vertical' | 'horizontal' = 'vertical';
+    @Input() detailSize: 'xs' | 'md' | 'lg' = 'lg';
+    @Input() detailItems = [];
```

Comment:
Es mala práctica ponerle nombres que hagan referencia a la implementación. El día de mañana se cambia la implementación interna y todos los inputs quedan en cumbia. 

```suggestion
    @Input() datos = [];
```

Reply:
Cambios hechos